credoai/reporting/dataset_fairness.py

```diff
@@ -205,66 +128,7 @@ def _plot_balance_metrics(self):
                     ax.legend_.set_title(label_name)
-            self.figs.append(f)
-
-    def _write_sensitive_feature_prediction(self):
-        content = f"""
-        #### Redundant Encoding
-    
-        <details open>
-        <summary>Assessment Description:</summary>
-        <br>
-        <p>The most important thing to check about your dataset is
-        "does it redundantly code a sensitive feature". Redundant encoding
-        means that the sensitive feature can be <em>reconstructed</em> from the features 
-        in your dataset. If it can be reconstructed, this means that your AI system
-        is implicitly trained on the sensitive feature, <em>even if it isn't explicitly included
-        in the dataset</em>.</p>
-
-        <p>To evaluate this, we train a model that tries to predict the sensitive feature from the
-        dataset. The score ranges from 0.5 - 1.0. If the score is 0.5, the model is random, and
-        no information about the sensitive feature is likely contained in the dataset. A value
-        of 1 means the sensitive feature is able to be perfectly reconstructed.</p>
-
-        <p>The <a href="#Feature-Balance>Feature Balance</a> and <a href="#Feature-Proxy-Detection>Feature Proxy Detection</a>
-        sections each provide additional perspective by diving into whether
-        individual features serve as proxies. Note that the overall dataset can be a 
-        proxy even if no individual feature is! That's where this score is important.</p>
-        
-        </details>
-
-        """
-        results_all = self.module.get_results()
-        sensitive_features = self.module.sensitive_features
-
-        for sensitive_feature_name in sensitive_features:
-            prefix = sensitive_feature_name + '-' 
-            score = results_all[prefix + 'sensitive_feature_prediction_score']
-            content += f"**Overall Proxy Score for {sensitive_feature_name}**: {score:.4f}" + " <br> "
-        
-        cell = (content, 'markdown')
-
-        return cell
-
-    def _write_group_diff(self):
-        cell = ("""
-                #### Feature Balance
-            
-                <details>
-                <summary>Assessment Description:</summary>
-                <br>
-                
-                <pr>Though potentially less important than balance of the
-                primary outcome, feature differences are also worth evaluating.</pr>
-
-                <pr>While some differences amongst groups should be expected, large deviations
-                are problematic. One of the main issues is that they may lead
-                to your dataset <em>redundantly encoding</em> sensitive features. In other
-                words, features that differ significantly between groups act as proxies
-                for the sensitive feature.</pr>
-                </details>
-                """, 'markdown')
-        return cell
+            self.figs.append(self._create_chart(f, BALANCE_DESCRIPTION))
```

Comment:
You meant `BALANCE_METRICS_DESCRIPTION`? `BALANCE_DESCRIPTION` does not seem to be defined anywhere.